### PR TITLE
Some updates for qemu build sctipt

### PR DESCRIPTION
**Pull request by nghiant2710**
The updates are
- Support building from private repository.
- upload arm and aarch64 binary in the same release.

### CODE REVIEW

File: automation/jenkins-build.sh [hunk 1]
```diff
@@ -1,4 +1,16 @@
 #!/bin/bash
+set -e
+set -o pipefail
+
+if [ -z "$isPrivate" ]; then
+	CURL_COMMAND="curl -SL https://codeload.github.com/$ACCOUNT/$REPO/tar.gz/v\$QEMU_VERSION -o qemu-\$QEMU_VERSION.tar.gz"
```
Comment (petrosagg) [inline automation/jenkins-build.sh]: Let's switch to a `git clone`. Jenkins has the right credentials already

Comment (Page-) [inline automation/jenkins-build.sh]: The docker build doesn't though @petrosagg

**Comment by nghiant2710** [comment]
@CameronDiver, can I have your review for this PR? We shouldn't hard-code QEMU version inside the dockerfile, we have parameters for QEMU version in jenkins build

File: automation/jenkins-build.sh (hunk 1 above)
Comment (CameronDiver) [inline automation/jenkins-build.sh]: Can't we forward the Jenkins credentials to the docker build?

**Pull request closed without merging**